src/packagedcode/npm.py

```diff
@@ -804,6 +804,46 @@ class NpmPackageLockJsonHandler(BaseNpmLockHandler):
     description = 'npm package-lock.json lockfile'
     documentation_url = 'https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json'
 
+    def parse(self, location):
```

Comment:
Why do you need to define another `parse()` function here and do nothing except logging? we already have one defined above.